packages/client/ui/Menu/Menu.tsx

```diff
@@ -0,0 +1,25 @@
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu'
+import React from 'react'
+import {twMerge} from 'tailwind-merge'
+
+interface MenuProps {
+  trigger: React.ReactNode
+  children: React.ReactNode
+}
+
+export const Menu = React.forwardRef<HTMLDivElement, MenuProps>(({trigger, children}, ref) => {
+  const contentClass = twMerge(
```

Comment:
+1 Current usage of `twMerge` is redundant, we are using it to be able to pass our own `className`. 
Same for `MenuItem`.

https://github.com/ParabolInc/parabol/blob/27a9efbbfd680febbe8f06203dab8e53e5e1fb56/packages/client/ui/Dialog/DialogClose.tsx#L13-L15

We also passing down props to support all radix props

https://github.com/ParabolInc/parabol/blob/27a9efbbfd680febbe8f06203dab8e53e5e1fb56/packages/client/ui/Dialog/DialogClose.tsx#L18

We can leave it as is and tweak later if the need arises to not spend more time on this now

Reply:
True - I've added the `className` now so twMerge is used 👍